app/build.gradle.kts

```diff
@@ -26,15 +26,15 @@ android {
     }
 
     compileSdk = 31
-    buildToolsVersion("31.0.0")
+    buildToolsVersion = "31.0.0"
 
     defaultConfig {
         applicationId = "app.attestation.auditor"
         minSdk = 26
         targetSdk = 31
         versionCode = 41
         versionName = versionCode.toString()
-        resConfigs("en")
+        resourceConfigurations.addAll(listOf("en"))
```

Comment:
This can simply use add instead of `addAll` since there's only one element.

Reply:
resourceConfigurations.add("en")

Reply:
Done